### PR TITLE
Add necessary repository documents

### PR DESCRIPTION
**Pull request by nicklathe**
- CONTRIBUTING.md
- LICENSE
- README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Terraform Controller
+====================
+
+Introduction
+------------
+
+This is a custom Kubernetes controller designed to run in the kube-system
```
Comment (dragan) [inline README.md]: Change this to the `scipian` namespace.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+namespace. It watches for changes on incoming CRD's and launches Jobs to 
+create/update/destroy infrastructure using Terraform.
+
+It is built with [Kubebuilder][kubebuilder], with full documentation found 
+[here][kubebuilder-book].
+
+[kubebuilder]: https://github.com/kubernetes-sigs/kubebuilder
+[kubebuilder-book]: https://book.kubebuilder.io/
+
+Running Locally
```
Comment (ncronquist) [inline README.md]: I copied your readme for the ssh-controller readme PR, so apparently I'm pretty ok with this as is :)

That being said, should we include any prerequisites here? Go version? Maybe a link to dep? Maybe that stuff is too basic for a Go project?

File: README.md (hunk 1 above)
Comment (nicklathe) [reply README.md]: This custom controller runs in the kube-system namespace.

File: README.md (hunk 2 above)
Comment (nicklathe) [reply README.md]: Maybe I explicitly point to the kubebuilder github for system requirements?

File: README.md (hunk 1 above)
Comment (nicklathe) [reply README.md]: We can of course change that, but this is the default for the manager.

Comment (dragan) [reply README.md]: Okay, it's fine for now then. For MVP we should have a work item to move it under the scipian namespace though.

File: README.md (hunk 2 above)
Comment (ncronquist) [reply README.md]: Oh good point. Sounds good.

**Comment by nicklathe** [comment]
@dragan ping.

**Pull request merged**